caffe2/core/tensor.h

```diff
@@ -117,6 +119,7 @@ class CAFFE2_API Tensor final {
    * The tensor will share the same instance (data, strides, sizes, etc) but
    * a different subset of APIs would be available
    */
+#if !defined(CAFFE2_IS_XPLAT_BUILD)
```

Comment:
What's going on here? These changes are not described in the PR body

Reply:
This excludes ATen/core/Tensor.h from the mobile build. It's weird we included it in the first place and it caused build errors for me because my added code depends on ATen things.